client/extensions/woocommerce/state/ui/shipping/zones/locations/selectors.js

```diff
@@ -567,7 +567,7 @@ export const getCurrentlyEditingShippingZoneLocationsList = createSelector(
 		const locations = getShippingZoneLocationsWithEdits( state, siteId, false );
 		return getShippingZoneLocationsListFromLocations( state, locations, maxCountries, siteId );
 	},
-	( state, maxCountries = 999, siteId = getSelectedSiteId( state ) ) => {
```

Comment:
Note that i needed to remove this initialization for the `no-unused-vars` not to fail.